metatensor-torch/include/metatensor/torch/block.hpp

```diff
@@ -59,11 +59,19 @@ class METATENSOR_TORCH_EXPORT TensorBlockHolder: public torch::CustomClassHolder
         return this->labels(0);
     }
 
-    /// get the len of the samples
+    /// Get the len of the samples
     int64_t len() const {
         return this->labels(0)->count();
     }
 
+    /// Get the shape of the values Tensor
+    torch::Tensor shape() const{
```

Comment:
this should be returned as the same type as `torch::Tensor::sizes`, which I think is either `at::IntArrayRef` or `at::IntArray`.

Reply:
which i cannot use, because it fails I think it is not TorchScriptable

Reply:
Yes, but you can use `this->values().sizes()` instead. 

`values_shape()` is here for integration with metatensor (which uses `vector<size_t>`)